sources/ss14-launcher-git.yaml

```diff
@@ -0,0 +1,3 @@
+type: git
+url: https://github.com/space-wizards/SS14.Launcher.git
```

Comment:
this has no reason to exist in a separate file.

Reply:
Yeah, I see what you mean. Makes it hard to figure out what version this is pointing to without inside knowledge :) I stuck together the two modules that were building from this source, and I changed the `update-ss14-version.py` script to simply read the current version from the flatpak manifest instead of trying to update it. (That's a good change anyway since it'll be easy to add an action to automate it, so I'm glad you pointed this out!).